build.sbt

```diff
@@ -67,20 +67,52 @@ lazy val `quill-sql` =
 lazy val `quill-sql-jvm` = `quill-sql`.jvm
 lazy val `quill-sql-js` = `quill-sql`.js
 
+val includeOracle =
+  sys.props.getOrElse("oracle", "false").toBoolean
+
+def includeIfOracle[T](t:T):Seq[T] =
+  if (includeOracle) Seq(t) else Seq()
+
 lazy val `quill-jdbc` =
   (project in file("quill-jdbc"))
     .settings(commonSettings: _*)
     .settings(mimaSettings: _*)
     .settings(
       fork in Test := true,
-      libraryDependencies ++= Seq(
-        "com.zaxxer"              % "HikariCP"             % "3.3.1",
-        "mysql"                   % "mysql-connector-java" % "5.1.47"             % Test,
-        "com.h2database"          % "h2"                   % "1.4.197"            % Test,
-        "org.postgresql"          % "postgresql"           % "42.2.5"             % Test,
-        "org.xerial"              % "sqlite-jdbc"          % "3.25.2"             % Test,
-        "com.microsoft.sqlserver" % "mssql-jdbc"           % "7.1.1.jre8-preview" % Test
-      )
+      resolvers ++= includeIfOracle( // read ojdbc7 jar in case it is deployed
```

Comment:
it seems that `quill-jdbc-monix` has the same code. Would it be possible to reuse?